nservicebus/upgrades/azure-functions-service-bus-isolated-3to4.md

```diff
@@ -1,9 +1,10 @@
 ---
-title: Azure Functions with Azure Service Bus Upgrade Version 2 to 3
-summary: How to upgrade Azure Functions with Azure Service Bus from version 2 to 3
+title: Azure Functions (isolated) for Service Bus Upgrade Version 3 to 4
+summary: How to upgrade Azure Functions (isolated worker) with Azure Service Bus from version 3 to 5
 component: ASBFunctions
-reviewed: 2021-10-09
+reviewed: 2021-12-16
 related:
+ - nservicebus/upgrades/7to8
 isUpgradeGuide: true
 upgradeGuideCoreVersions:
  - 8
```

Comment:
Functions versions 2 and 3 are both NServiceBus Core 7, right?
```suggestion
 - 7
```

Reply:
No, in-process worker v3 targets core v8 , see https://github.com/Particular/NServiceBus.AzureFunctions.InProcess.ServiceBus/blob/master/src/NServiceBus.AzureFunctions.InProcess.ServiceBus/NServiceBus.AzureFunctions.InProcess.ServiceBus.csproj#L18

Reply:
my bad, you are correct again